lib/hammer_cli_foreman_remote_execution/job_invocation.rb

```diff
@@ -202,12 +221,9 @@ def self.extend_data(invocation)
         invocation['randomized_ordering'] = targeting['randomized_ordering']
 
         hosts = targeting['hosts'].map do |host|
-          host_item = " - #{host['name']}"
-          host_item << ", Job status: #{host['job_status']}" if host['job_status']
-          host_item
+          { 'Name' => host['name'], 'Job status' => host['job_status'], 'Inputs' => host[:inputs] }.compact
```

Comment:
Not sure if show empty fields when status/input options are on or remove empty key:values